frontend/src/pages/login.tsx

```diff
@@ -185,6 +190,7 @@ export default function Login() {
           {options?.local && (
             <form
               ref={formRef}
+              onSubmit={handleSubmit}
```

Comment:
I would have defined handleSubmit function here anonymously instead of named const above, but this difference is pedantic.